apps/breethe_web/config/config.exs

```diff
@@ -42,8 +42,7 @@ config :sentry,
   environment_name: Mix.env(),
   enable_source_code_context: true,
   root_source_code_path: File.cwd!(),
-  included_environments: [:prod],
-  json_library: Poison
```

Comment:
Is this no longer necessary (or advisable)?

Reply:
Yep, we don't need it - sentry now uses a different library to serialize the JSON

Reply:
ah this is for Sentry 🤦‍♂